web/pgadmin/tools/sqleditor/static/js/components/sections/Query.jsx

```diff
@@ -72,8 +71,15 @@ export default function Query({onTextSelect}) {
   const lastCursorPos = React.useRef();
   const pgAdmin = usePgAdmin();
   const preferencesStore = usePreferences();
-
   const queryToolPref = queryToolCtx.preferences.sqleditor;
+  const [isEditorDisabled, setIsEditorDisabled] = useState(false);
+  const isqueryToolAndUrl = queryToolCtx.params.is_query_tool && queryToolCtx.params.query_url;
+
+  useEffect ( () => {
+    if (queryToolCtx.preferences?.sqleditor?.copy_sql_to_query_tool || queryToolCtx.params.is_query_tool && queryToolCtx.params.query_url!== '') {
+      setIsEditorDisabled(true);
```

Comment:
where is it enabled again? It should enable again once the script it loaded.

Reply:
it enables when the doc updated successfully.